src/js/components/SelectMultiple/index.d.ts

```diff
@@ -5,7 +5,7 @@ export interface SelectMultipleProps extends BasicSelectProps {
   defaultValue?: (string | number | object)[];
   help?: string | React.ReactNode;
   limit?: number;
-  value?: (string | number | object)[];
+  value?: string | number | boolean | JSX.Element | object;
```

Comment:
Why are we including boolean and JSX.Element here?

Reply:
I added the JSX.Element because it was there in Select Value type.

Reply:
https://v2.grommet.io/selectmultiple#value

we dont need boolean here

Reply:
It also doesn't look like JSX.Element is a supported SelectMultiple value